### DISCUSSION TITLE
Kavanozdaki sinekler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#308] [Zeka Sorusu]
Kapağı sıkıca kapalı bir kavanozun içinde bir miktar toprak ve iki de sinek var.
Sinekler kavanozun dibinde yerdeler. Son derece hassas elektronik terazi ile ölçüm yapıyoruz. Çıkan ağırlığı kenara yazıyoruz.
İkinci ölçümü yaptığımız anda sinekler kavanozun içinde uçuyorlar.
İki ölçüm arasında fark olur mu?

**Comment by Burak2210** [comment]
Olmaz çünkü sinekler uçtuğu sırada aynı zamanda ağırlıkları kadar yere hava basıncı yaparlar böylece ağırlık aynı çıkar

**Comment by Mertcakm** [comment]
Fark olmaz çünkü terazi kütle ölçer ve kütle madde miktarıdır sinek havalandığında madde miktarı değişmeyeceği için iki ölçüm arasında fark bulunmaz.

**Reply by sonsuzus** [reply]
Cevaplanmamış sorulara bakalım.
